activesupport/lib/active_support/time_with_zone.rb

```diff
@@ -303,6 +303,13 @@ def +(other)
         result = utc + other
         result.in_time_zone(time_zone)
       end
+    rescue TypeError
+      ActiveSupport.deprecator.warn(
+        "Adding an instance of #{other.class} to an instance of #{utc.class} is deprecated. This behavior will raise " \
```

Comment:
(non-blocking)

```suggestion
        "Adding an instance of #{other.class} to an instance of #{self.class} is deprecated. This behavior will raise " \
```

The user wasn't adding to `utc.class` (which is always Time) but `TimeWithZone`, I believe adding two `Time` together will already raise (which is how we get here).